src/main/java/core/basesyntax/Main.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import core.basesyntax.thread.MyRunnable;
+import core.basesyntax.thread.MyThread;
+
 public class Main {
     public static void main(String[] args) {
-        // write your code here
+        Thread myThread = new MyThread();
```

Comment:
I'd rather named it "myDaemonThread" or somehow else to show that it's our daemon thread